src/main/java/uk/gov/hmcts/reform/bulkscan/payment/processor/config/QueueClientsConfiguration.java

```diff
@@ -10,7 +10,7 @@
 import org.springframework.context.annotation.Profile;
 
 @Configuration
-@Profile("!nosb") // do not register handler for the nosb (test) profile
+@Profile("!functional") // do not register handler for the nosb (test) profile
```

Comment:
This change will make `QueueClientsConfiguration` available when `nosb` profile is used.
Was that your intention?

Reply:
There will be no nosb profile - it is renamed to functional.

Reply:
it would be worth stating that in the PR title on description.

Reply:
Have added